src/snyk/common/api/headers.ts

```diff
@@ -0,0 +1,4 @@
+export const DEFAULT_API_HEADERS: Readonly<Record<string, string | boolean>> = {
+  Accept: 'application/json',
```

Comment:
For consistency, I'd prefer to have single quotes on both headers. Do we need a user agent or referrer header for tracking?

Reply:
Maybe it would also be helpful good to add `'Accept-Encoding': 'gzip'`

Reply:
>Do we need a user agent or referrer header for tracking?

No, so far we were not sending any.

I'll add the accept-encoding there. 👍

Reply:
Though can't have quotes on both sides as it violates this rule: https://eslint.org/docs/rules/quote-props